src/app/pages/ingresos/components/lista-ingresos/lista-ingresos.component.ts

```diff
@@ -15,6 +18,11 @@ import { cargarTipoIngreso } from '../../actions/ingresos.actions';
 import { TranslateService } from '@ngx-translate/core';
 import { TranslateFormItemsService } from '../../../../shared/helpers/translate-form-items.service';
 import { DATOS_INGRESOS } from '../../../../../assets/mock/tiposIngresos';
+import { IngresoService } from '../../../../shared/services/ingresos/ingreso.service';
+import { Ingreso } from '../../../../shared/services/ingresos/dto/ingreso.dto';
+import { numeric } from '@rxweb/reactive-form-validators';
+import Swal from 'sweetalert2';
+import { PopUpManager } from '../../../../@core/managers/popUpManager';
```

Comment:
```suggestion
import { PopUpManager } from '../../../../@core/managers/popUpManager';
```
En lo posible, evitar las dependencias directas de `Swal`: El `PopUpManager` internamente abstrae el uso de `Swal`. Si es necesario mostrar avisos en cascada, hacer que el `PopUpManager` retorne el evento Swal